src/redux/reducers/ui.ts

```diff
@@ -222,6 +222,12 @@ export const uiSlice = createSlice({
     closeFolderExplorer: (state: Draft<UiState>) => {
       state.folderExplorer = {isOpen: false};
     },
+    closeKubeConfigBrowseSetting: (state: Draft<UiState>) => {
+      state.kubeConfigBrowseSetting = {isOpen: true};
+    },
+    openKubeConfigBrowseSetting: (state: Draft<UiState>) => {
+      state.kubeConfigBrowseSetting = {isOpen: false};
```

Comment:
Same as above

Reply:
If you want to suggest me better naming convention then please do. (like instead of `isOpen` to `isClose` or any other)

I believe it is set the state, to check whether to close the `Drawer` or not. You can  find the relevant code in `App.tsx` component.

Reply:
You don't have to pass it from child to parent, just create a new property on redux store and use it across all the components

Reply:
Similar to `kubeConfigBrowseSetting`

Reply:
okay